so_pysm_models/co_lines.py

```diff
@@ -0,0 +1,207 @@
+import numpy as np
+
+import healpy as hp
+
+
+class COLines:
+    def __init__(
+        self,
+        target_nside,
+        has_polarization=True,
+        line="10",
+        include_high_galactic_latitude_clouds=False,
+        polarization_fraction=0.001,
+        theta_high_galactic_latitude_deg=0.,
```

Comment:
@giuspugl shouldn't the default for theta be higher?

Reply:
yes I agree! we can set the default to  20 deg .